package.json

```diff
@@ -6,39 +6,48 @@
     "node": "8.x.x"
   },
   "scripts": {
-    "start": "sh scripts/start.sh",
-    "dev": "sh scripts/dev.sh",
-    "test": "sh scripts/test.sh",
-    "test:watch": "yarn jest -- --watch --runInBand",
     "assets": "sh scripts/assets.sh",
     "deploy": "sh scripts/deploy.sh",
+    "dev": "sh scripts/dev.sh",
+    "precommit": "lint-staged",
+    "prepush": "yarn run type-check",
+    "prettier": "prettier",
+    "prettier-write": "yarn run prettier --write",
     "mocha": "sh scripts/mocha.sh",
-    "task": "node -r coffeescript/register -r @babel/register -r dotenv/config"
+    "start": "sh scripts/start.sh",
+    "task": "node -r coffeescript/register -r @babel/register -r dotenv/config",
+    "test:watch": "yarn jest -- --watch --runInBand",
+    "test": "sh scripts/test.sh",
+    "type-check": "tsc --pretty --noEmit"
   },
   "resolutions": {
     "babel-core": "^7.0.0-0",
-    "babel-plugin-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/babel-plugin-relay-1.5.0-artsy.5.tgz",
-    "relay-compiler": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/relay-compiler-1.5.0-artsy.5.tgz",
-    "relay-runtime": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/relay-runtime-1.5.0-artsy.5.tgz",
-    "react-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/react-relay-1.5.0-artsy.5.tgz"
+    "babel-plugin-relay":
+      "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/babel-plugin-relay-1.5.0-artsy.5.tgz",
+    "relay-compiler":
+      "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/relay-compiler-1.5.0-artsy.5.tgz",
+    "relay-runtime":
+      "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/relay-runtime-1.5.0-artsy.5.tgz",
+    "react-relay":
+      "https://github.com/alloy/relay/releases/download/v1.5.0-artsy.5/react-relay-1.5.0-artsy.5.tgz"
   },
   "dependencies": {
     "@artsy/express-reloadable": "^1.3.0",
     "@artsy/palette": "^2.9.1",
     "@artsy/reaction": "^1.17.5",
-    "@babel/cli": "7.0.0-beta.42",
-    "@babel/core": "7.0.0-beta.42",
-    "@babel/node": "7.0.0-beta.42",
-    "@babel/plugin-proposal-class-properties": "7.0.0-beta.42",
-    "@babel/plugin-proposal-decorators": "^7.0.0-beta.42",
-    "@babel/plugin-transform-runtime": "7.0.0-beta.42",
-    "@babel/preset-env": "7.0.0-beta.42",
-    "@babel/preset-es2015": "7.0.0-beta.42",
-    "@babel/preset-react": "7.0.0-beta.42",
-    "@babel/preset-stage-3": "7.0.0-beta.42",
-    "@babel/preset-typescript": "7.0.0-beta.42",
-    "@babel/register": "7.0.0-beta.42",
-    "@babel/runtime": "7.0.0-beta.42",
+    "@babel/cli": "7.0.0-beta.54",
+    "@babel/core": "7.0.0-beta.54",
+    "@babel/node": "7.0.0-beta.54",
+    "@babel/plugin-proposal-class-properties": "7.0.0-beta.54",
+    "@babel/plugin-proposal-decorators": "7.0.0-beta.42",
```

Comment:
Why is plugin-proposal-decorators not bumped as well?  have a version running locally that seems ok on beta.54

Reply:
I _think_ this was deliberate because tests would fail but let me bump and try again.

Reply:
This should work if you add `["@babel/plugin-proposal-decorators", {"legacy": true}],` to `.babelrc`

Reply:
I think it also has to come first in the plugins array, or maybe just before `plugin-proposal-class-properties`